tsconfig.json

```diff
@@ -4,11 +4,16 @@
     "emitDeclarationOnly": false,
     "incremental": false,
     // Required to run benchmark command from root directory
-    "typeRoots": ["node_modules/@types", "./types"],
+    "typeRoots": [
+      "node_modules/@types",
+      "./types",
+      "${configDir}/node_modules/@types",
+      "${configDir}/types"
+    ],
+    "noCheck": false,
```

Comment:
For building we can skip type checks as we have seperate task for `check-types`. Saved 30% of build time on my machine. 

https://devblogs.microsoft.com/typescript/announcing-typescript-5-6/#the---nocheck-option